app/components/ui/tale-tab-metadata/template.hbs

```diff
@@ -7,9 +7,38 @@
     </div>
 
     <div class="inline fields ui grid">
-        <label class="two wide column right aligned">Authors</label>
+      <label class="two wide column right aligned">Authors</label>
+    </div>
+    {{#each taleAuthors as |author|}}
+    <div class="inline fields ui grid">
+      <label class="two wide column right aligned"></label>
+      <div class="field thirteen wide column" id=author.id>
+        <div class="ui form">
+          <div class="inline fields">
+            <label>First Name:</label>
+              <div class="field">
+                {{input type="text" name='first-name' value=author.firstName placeholder="Sur Name" }}
+              </div>
+            <label>Last Name:</label>
+            <div class="field">
+              {{input type="text" name='last-name' value=author.lastName placeholder="Family Name" }}
+            </div>
+          <label>ORCID:</label>
+          <div class="field">
+            {{input type="url" name='orcid-id' placeholder="https://orcid.org/0000-0002-1825-0097"  value=author.orcid }}
+          </div>
+          <i class="fa fa-window-close" aria-hidden="true" style="color: black" {{action 'removeAuthor' author.id}}></i>
+        </div>
+      </div>
+    </div>
+  </div>
+  {{/each}}
+      
+    <div class="inline fields ui grid">
+        <label class="two wide column right aligned"></label>
         <div class="field thirteen wide column">
-            {{input value=model.tale.authors readonly=cannotEditTale}}
+            <label></label> <i class="fas fa-plus" style="color: #2185d0; margin-left: 10%;"{{action 'addNewAuthor'}}></i>
+            <span style="color: #2185d0;" {{action 'addNewAuthor'}}>&nbsp;Add another author</span>
```

Comment:
This button (as well as any existing authors' `inputs`) should be disabled if the user doesn't have write access to the Tale.

You should just need to add `readonly=cannotEditTale` to the `{{input}}` components, but you may need to change from using a `<span>` to a `<button>`.. I'm unsure if `<span>` actually has a `disabled` attribute